classes/booking.js

```diff
@@ -1,7 +1,12 @@
 class Booking {
-    constructor() {
-
-    }
+  constructor(booking) {
+    //dont use object.assign? bad code practice?
```

Comment:
I'm curious to see how you would use object.assign in this?  I've only ever used it once. But our booking class looks pretty much the same.

Reply:
In theory since object.assign would assign all the keys based on the input object I could skip the process of assigning values with an identical name as they are on the input object.

Apparently this is a bad move though as it's not type safe, so a habit I figure it's probably best just to avoid
https://stackoverflow.com/questions/55583732/what-is-the-purpose-of-object-assign-in-the-constructor-of-a-typescript-object/55583853